web/layout.go

```diff
@@ -93,6 +97,19 @@ func (r *LayoutRender) addFileFromFS(templatesFS fs.FS, file string) {
 		"sum": func(a int, b int) int {
 			return a + b
 		},
+		"markup": func(md string) template.HTML {
+			extensions := parser.CommonExtensions | parser.AutoHeadingIDs
+			p := parser.NewWithExtensions(extensions)
+
+			htmlFlags := html.CommonFlags | html.HrefTargetBlank
+			opts := html.RendererOptions{Flags: htmlFlags}
+			renderer := html.NewRenderer(opts)
+
+			// html := markdown.ToHTML([]byte(md), nil, nil) // could be as simple as this
+			html := markdown.ToHTML([]byte(md), p, renderer)
+			result := template.HTML(html)
+			return result
```

Comment:
The `html` variable here collides with the imported `github.com/gomarkdown/markdown/html` package.
Maybe rename it as follows?
```suggestion
			rendered := markdown.ToHTML([]byte(md), p, renderer)
			result := template.HTML(rendered)
			return result
```